game.js

```diff
@@ -296,6 +313,26 @@ class Game extends Screen {
 		text(this.score, width / 2, LINE_SPACING)
 	}
 
+	drawTargetScore() {
+		textAlign(CENTER, CENTER)
+		textSize(30)
+		fill(255, 255, 250)
+		strokeWeight(4)
+		this.calculatePointsToGo()
+		text('You need a total of ' + targetScore + ' points to win.', width / 2, LINE_SPACING * 13)
+		text('Only ' + (this.calculatePointsToGo()) + ' points to go!', width / 2, LINE_SPACING * 14)
+	}
+
+	calculatePointsToGo(pointsToGo) {
```

Comment:
Why does this function need a parameter `pointsToGo`?  You don't use it in the calculation.